src/liboslexec/llvm_instance.cpp

```diff
@@ -782,12 +784,18 @@ BackendLLVM::build_llvm_init ()
 {
     // Make a group init function: void group_init(ShaderGlobals*, GroupData*)
     // Note that the GroupData* is passed as a void*.
-    std::string unique_name = Strutil::sprintf ("__direct_callable__group_%d_init", group().id());
+    std::string unique_name = Strutil::sprintf ("__direct_callable__group_%s_%d_init",
```

Comment:
`unique_name` is only used in the block below I think -- can it be moved inside there?

Reply:
Oh nevermind, I see its used for `make_function` as well.